packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -180,211 +203,227 @@ It took ${timeInMsToDownloadAll}ms to execute download() for all binaryTargets.`
+            "size": 19440080,
+          },
+          {
+            "name": "migration-engine-rhel-openssl-3.0.x",
+            "size": 19441160,
+          },
+          {
+            "name": "migration-engine-windows.exe",
+            "size": 15780864,
+          },
+          {
+            "name": "query-engine-darwin",
+            "size": 17840936,
+          },
+          {
+            "name": "query-engine-darwin-arm64",
+            "size": 16655984,
+          },
+          {
+            "name": "query-engine-debian-openssl-1.0.x",
+            "size": 19287104,
+          },
+          {
+            "name": "query-engine-debian-openssl-1.1.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-debian-openssl-3.0.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-1.0.x",
+            "size": 16722600,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-1.1.x",
+            "size": 17462856,
+          },
+          {
+            "name": "query-engine-linux-arm64-openssl-3.0.x",
+            "size": 18788592,
+          },
+          {
+            "name": "query-engine-linux-musl",
+            "size": 16664640,
+          },
+          {
+            "name": "query-engine-linux-musl-arm64-openssl-1.1.x",
+            "size": 17708728,
+          },
+          {
+            "name": "query-engine-linux-musl-arm64-openssl-3.0.x",
+            "size": 19120392,
+          },
+          {
+            "name": "query-engine-linux-musl-openssl-3.0.x",
+            "size": 16775312,
+          },
+          {
+            "name": "query-engine-linux-static-arm64",
+            "size": 15984368,
+          },
+          {
+            "name": "query-engine-linux-static-x64",
+            "size": 19274912,
+          },
+          {
+            "name": "query-engine-rhel-openssl-1.0.x",
+            "size": 19287104,
+          },
+          {
+            "name": "query-engine-rhel-openssl-1.1.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-rhel-openssl-3.0.x",
+            "size": 16808072,
+          },
+          {
+            "name": "query-engine-windows.exe",
+            "size": 19483648,
+          },
+          {
+            "name": "query_engine-windows.dll.node",
+            "size": 17259520,
+          },
+        ]
+      `)
 
       expect(await getVersion(queryEnginePath, BinaryType.QueryEngineBinary)).toMatchInlineSnapshot(
-        `"query-engine eac182fd33c63959a61946df56831625a9a39627"`,
+        `"query-engine b20ead4d3ab9e78ac112966e242ded703f4a052c"`,
       )
       expect(await getVersion(migrationEnginePath, BinaryType.MigrationEngineBinary)).toMatchInlineSnapshot(
-        `"migration-engine-cli eac182fd33c63959a61946df56831625a9a39627"`,
+        `"schema-engine-cli b20ead4d3ab9e78ac112966e242ded703f4a052c"`,
```

Comment:
Isn't that a P5 change?

Reply:
This specifically was already changed on the engine side in April (https://github.com/prisma/prisma-engines/pull/3844). We actually have a hack in `prisma version` command that replaces `schema-engine-cli` with `migration-engine-cli` in the output. But here it's the raw command output.

Reply:
Ok, so we were testing an version older than the one from April until now? Sounds suboptimal. Can't we use the engine hash of the current version in this test?

Reply:
Changed `FIXED_ENGINE_HASH` to `CURRENT_ENGINE_HASH` in all tests except those that snapshot engine size.

Reply:
And now, 13 minutes later I understand that this will lead to small size changes each commit which will make the snapshots fail I think. Sorry @aqrln.

Reply:
No no, all good, there's no problem. The tests that snapshot engine sizes still use the fixed commit hash that needs to be updated manually. I only changed the rest of the tests (including this one).

Reply:
ah not, wait, I'm wrong, I thought it was a separate test but it's a part of the test that snapshots sizes :/

Reply:
somehow I also didn't commit all changes in the first place

Reply:
also the commit hash would've been changing here and failing snapshots, so it wouldn't work even if it was a separate test which I also failed to realise — probably time to wrap up for today 😄